batavia/modules/time.js

```diff
@@ -23,3 +23,75 @@ batavia.modules.time = {
         }
     }
 };
+
+batavia.modules.time.struct_time = function (sequence) {
+    /*
+        copied from https://docs.python.org/2/library/time.html#time.struct_time
+        WHAT ATTRIBUTION DOES THIS NEED?
```

Comment:
No copyright attribution is required, because you're not "copying" anything. You're not re-using an implementation or algorithm. You're just following a format specification. So - it's worth pointing at the documentation that defines that specification (although it would be better to point at the Python 3.4 docs, rather than Python 2), but you don't have to acknowledge copyright in this case.